docs/paid-features/activating-the-enterprise-edition.md

```diff
@@ -23,18 +27,36 @@ You can either:
 - [Download the latest metabase-enterprise JAR](https://downloads.metabase.com/enterprise/latest/metabase.jar) (the filename is the same, irrespective of your plan), or
 - [Get the latest Docker image](https://hub.docker.com/r/metabase/metabase-enterprise/) at `metabase/metabase-enterprise:latest`.
 
-Run Metabase as you would normally, then go to __Settings__ > __Admin settings__, and click __License and Billing__ in the lefthand sidebar. Paste in your license token under __License__ and click __Activate__.
+Run Metabase as you would normally, then go to **Settings** > **Admin settings**, and click **License and Billing** in the lefthand sidebar. Paste in your license token under **License** and click **Activate**.
 
 ## **Validating your token**
 
-Your Metabase needs to be able to access the internet (specifically `https://store.metabase.com/api/[token-id]/v2/status`) in order to validate the token and maintain access to the advanced features.
+To validate your token and maintain access to paid features, your Metabase needs to be able to access the Internet, specifically:
+
+```
+https://token-check.metabase.com/api/[token-id]/v2/status
+```
+
+(substituting `[token-id]` with your token ID).
+
+If your Metabase can't validate the token, it'll disable the paid features, but will continue to work normally as if you were running the Open Source edition.
 
-If your Metabase can't validate the token, it'll disable the advanced features, but will continue to work normally otherwise, as if it were the Open Source edition.
+## Routing outbound Metabase traffic through a proxy
 
 In case you need to route outbound Metabase traffic through a proxy on your network, use the following command when starting Metabase:
 
 ```
 java -Dhttps.proxyHost=[your proxy's hostname] -Dhttps.proxyPort=[your proxy's port] -jar enterprise_metabase.jar
 ```
 
-Depending on your organization’s setup, you may need to take additional configuration steps. If the command above doesn't work for you, we recommend reaching out to your internal infrastructure or dev ops teams for assistance.
\ No newline at end of file
+Depending on your organization’s setup, you may need to take additional configuration steps. If the command above doesn't work for you, we recommend reaching out to your internal infrastructure or dev ops teams for assistance.
+
+## IP addresses to whitelist
+
+If you're hosting Metabase behind a firewall, you'll need to whitelist these IP addresses to ensure your token checks can phone home to verify your license.
```

Comment:
This sentence should specifically say it's for the domain `token-check.metabase.com`.
```suggestion
If you're hosting Metabase behind a firewall that blocks outgoing connections, you'll need to allow these IP addresses to ensure access to `token-check.metabase.com` to verify your license.
```